saas_portal_start/static/src/js/main.js

```
@@ -36,7 +36,6 @@ $(document).ready(function() {
             $input.attr('data-content', "Please choose your domain name");
             error = true;
         }
-        db_name = _.str.slugify(db_name);
```

Why?

---

because it removes dots

---

Using dots change the idea of module (quick start on a sub domain). It should be optional or in another module

---

Ok, I can put back this ``slugify`` thing. But for me, it is not friendly user interface just to silently replace dots that a user for some reason wants to be there

---

I don't see either how dots change the idea of this module

---

A user can quickly start on a subdomain regardless of any dots in the input field. With dots or without dots - it doesn't matter at all

---

Slugify is not for dots removing. It changes any non ascii symbols

---

It removes dots. Nonascii symbols are detected by regex anyway